cfme/services/catalogs/service_catalogs.py

```diff
@@ -73,4 +76,5 @@ def order_stack_item(self, catalog, catalog_item):
                                     'catalog_item': catalog_item})
         stack_form.fill(self.stack_data)
         sel.click(form_buttons.submit)
+        sleep(5)
```

Comment:
sleeps tend to be a problem due to
a) unconditionally making tests take longer
b) creating fragility since the test may break

please use a wait_for if it takes a few tries until the message appears